src/odemis/util/img.py

```diff
@@ -777,3 +777,18 @@ def _getBoundingBox(content):
         md_pos[1] - half_shape_wc[1],
     )
     return rect
+
+
+def getOutliers(data, outliers=0):
+    """
+    Calculates the percentage of outliers that should be discarded.
```

Comment:
As it doesn't return a percentage, this sentence is a little miss-leading. I'd suggest: finds the minimum and maximum values when discarding a given percentage of outliers.